compiler/rustc_const_eval/src/interpret/operand.rs

```diff
@@ -186,27 +186,26 @@ pub struct ImmTy<'tcx, Prov: Provenance = CtfeProvenance> {
 impl<Prov: Provenance> std::fmt::Display for ImmTy<'_, Prov> {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         /// Helper function for printing a scalar to a FmtPrinter
-        fn p<'a, 'tcx, Prov: Provenance>(
-            cx: &mut FmtPrinter<'a, 'tcx>,
+        fn g<'a, 'tcx, Prov: Provenance>(
```

Comment:
Maybe let's rename this to `print_scalar` or so if you're renaming the function anyway.

Reply:
ok